src/components/notes/editor-view.jsx

```diff
@@ -1,18 +1,18 @@
-import React, { useCallback, useRef, useEffect } from 'react'
+import React, { useCallback, useRef, useEffect, useMemo } from 'react'
 
 import { Editor, WithEditorActions } from '@atlaskit/editor-core'
 
 import { MainTitle } from 'cozy-ui/react/Text'
 import Textarea from 'cozy-ui/react/Textarea'
-import { translate } from 'cozy-ui/react/I18n'
+import { useI18n } from 'cozy-ui/react/I18n'
 import useEventListener from 'cozy-ui/react/hooks/useEventListener'
 
 import editorConfig from 'components/notes/editor_config'
 import HeaderMenu from 'components/header_menu'
 import styles from 'components/notes/editor-view.styl'
 
 function updateTextareaHeight(target) {
-  target.style.height = `${target.scrollHeight}px`
+  if (target) target.style.height = `${target.scrollHeight}px`
```

Comment:
does that means that `titleEl.current` can be null or undefined? how can it happen ?

Reply:
C'est la seule modif vraiment honteuse : Je n'ai pas réussi à faire fonctionner les attributs ref lors du mount par Enzime, et cette condition est là pour compenser la ref DOM qui n'est pas là dans ce cas.